### PR TITLE
Add oikotie.fi and tori.fi to Schibsted. Remove Tori as a separate entry.

### PR DESCRIPTION
**Pull request by momentti**
Both sites use Schibsted account to log in.

### CODE REVIEW

**Comment by phallobst** [comment]
Since `Schibsted` has no documentation link, but `Tori` has (or had), I wonder if we should include that url here. It describes the common login backend, after all.

**Comment by momentti** [comment]
I added FAQ page as documentation. It used to be behind login, but not anymore.

File: entries/s/schibsted.com.json [hunk 1]
```diff
@@ -24,12 +24,15 @@
...
-      "tek.no"
+      "tek.no",
+      "oikotie.fi",
+      "tori.fi"
     ],
     "tfa": [
       "sms",
       "totp"
     ],
+    "documentation": "https://login.schibsted.com/profile-pages/faq",
```
Comment (Carlgo11) [inline entries/s/schibsted.com.json]: The entire page is rather long and it is primarily this part that's of interest.
```suggestion
    "documentation": "https://login.schibsted.com/profile-pages/faq#how-to-activate-and-deactivate-2fa",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**